packages/prettier/source/index.ts

```diff
@@ -1,3 +1,19 @@
-export function hello() {
-    console.log("Hello from @adchitects/prettier-config!");
-}
+import { createMergedConfig } from "@workspace/shared/configuration";
+import { hasModule } from "@workspace/shared/module";
+
+import prettier from "./prettier.js";
+
+import phpPlugin from "./plugins/php.js";
+
+const mergedConfig = createMergedConfig([
+	// Base
+	prettier,
+
+	// Plugins
+    // TODO: @acwo I need your help to decide how to help detect when the project needs this plugin.
```

Comment:
@acwo 

This is where I will need your help.

Reply:
@xeho91 

> // TODO: @acwo I need your help to decide how to help detect when the project needs this plugin.
> // Is there any specific Node.js package which is ALWAYS used with PHP projects?

- From what I see we could add some specific keywords to PHP projects' `package.json` file, for example:
`"keywords": [adchitects-php-project]`
- We could add this keyword to our updated version of `wp-starter-pack` and all further projects based on it would have the keyword.
- Can we use that? Will this allow us to detect when the project needs prettier PHP plugin?

Reply:
Clever idea!💡

I will prepare the shared helper function `hasKeywords(["array",  "of", "keywords"])` from for you tomorrow.